modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/index.js

```diff
@@ -185,6 +191,36 @@ const InviteTeamMembersPage = ({
 
 		if (filledEmails.length) {
 			setIsLoadingUserInvitation(true);
+
+			const filledEmailsPromises = filledEmails.map(
```

Comment:
Also, the addTeamMemberInvitation needs to be from filledEmailsPromises. Because still a issue if one of 3 don't happen